### PR TITLE
TRIVIAL: Changelog for 5.3 release

### PR DESCRIPTION
**Pull request by peterlacko**

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/1095/ : SUCCESS in 8s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/612/ : SUCCESS in 2m 08s

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,10 @@
+# Changelog
+## 5.3.0 (August 23, 2018)
+- Added Heatmap chart ([commit](https://github.com/gooddata/gooddata-react-components/commit/660b01d949e08623f2cc1b3e7a091ca16efaf29e))
+- Improve lebels rendering of Treemap ([commit](https://github.com/gooddata/gooddata-react-components/pull/440/commits/5d5d002128e15f31e2af27b2ccccec4a809db0c9))
```
Comment (evgeniabogranskaya) [inline CHANGELOG.md]: Improved label rendering in Treemaps

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/1096/ : SUCCESS in 6s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/613/ : SUCCESS in 1m 32s

**Comment by gdgate** [comment]
Build failed (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-merge-compliance-check/216/ : SUCCESS in 11s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/629/ : SUCCESS in 3m 17s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/342/ : SUCCESS in 9m 47s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-examples-zuul-docker/111/ : FAILURE in 12m 35s

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-merge-compliance-check/217/ : SUCCESS in 16s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/630/ : SUCCESS in 2m 09s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/343/ : SUCCESS in 8m 28s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-examples-zuul-docker/112/ : SUCCESS in 10m 19s

**Pull request merged**